nginx.conf

```diff
@@ -24,6 +24,21 @@ server {
         proxy_set_header X-Forwarded-Host $host;
     }
 
+    # Proxy /api/media
+    location /api/media {
+        rewrite ^/api/media/(.*) /$1 break;
+        proxy_set_header Host media-minio:9000;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_connect_timeout 300;
+        # Default is HTTP/1, keepalive is only enabled in HTTP/1.1
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
```

Comment:
`Connection ""`?

Reply:
Not sure what MinIO uses this header for, however this is what they specified in the example: https://min.io/docs/minio/linux/integrations/setup-nginx-proxy-with-minio.html